internal/cmd/internal/olmv1/catalog_create.go

```diff
@@ -13,7 +13,7 @@ import (
 
 // NewCatalogCreateCmd allows creating a new catalog
 func NewCatalogCreateCmd(cfg *action.Configuration) *cobra.Command {
-	i := v1action.NewCatalogCreate(cfg.Client)
+	i := v1action.NewCatalogCreate(cfg)
```

Comment:
Good catch. I have gone through the code many times but did not see this.

Reply:
I didn't realize it at first either, I stumbled on this when I checked the implementation for the other subcommands within kubectl-operator